app/code/Magento/Theme/view/frontend/templates/html/header/logo.phtml

```diff
@@ -18,8 +18,8 @@
     <a class="logo" href="<?= $block->getUrl('') ?>" title="<?= /* @escapeNotVerified */ $storeName ?>">
 <?php endif ?>
         <img src="<?= /* @escapeNotVerified */ $block->getLogoSrc() ?>"
-             title="<?= /* @escapeNotVerified */ $block->getLogoAlt() ?>"
-             alt="<?= /* @escapeNotVerified */ $block->getLogoAlt() ?>"
+             title="<?= /* @escapeNotVerified */ $block->escapeHtmlAttr($block->getLogoAlt()) ?>"
+             alt="<?= /* @escapeNotVerified */  $block->escapeHtmlAttr($block->getLogoAlt()) ?>"
```

Comment:
`escapeNotVerified` must be changed accordingly.

Reply:
removed them